src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -114,6 +116,9 @@ public function load(array $configs, ContainerBuilder $container)
         // load ACL
         if (isset($config['acl'])) {
             $this->aclLoad($config['acl'], $container);
+        } else {
+            $container->removeDefinition(InitAclCommand::class);
```

Comment:
broken if console not available, should go in the previous if (console.xml loading) and acl loading be moved just before console loading